src/api/app/views/webui/image_templates/index.html.haml

```diff
@@ -13,9 +13,9 @@
             %dd
               %span.form-radio
               - if template.has_icon?
-                %img{ alt: template.title, src: template.source_path('_icon'), width: 64, height: 64 }
+                %img{ alt: template.title, src: template.source_path('_icon') }
               - else
-                = image_tag('distributions/kiwi.png', width: 64, height: 64)
+                = image_tag('image-template.png')
```

Comment:
But the problem is that it hasn't got the same size, that's why we were not using it here. I would say that I need the image in a bigger size and add it also to the sprite.

![image](https://cloud.githubusercontent.com/assets/16052290/20706550/223b61d4-b628-11e6-9aa3-8bead4517654.png)